File: Java/Student Grade Manager/src/Student.java
public class Student {
    private String name;
    private Score[] studentScores;

    public Student(String name, int numberOfSubjects) {
        this.name = name;
        this.studentScores = new Score[numberOfSubjects];

    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Score getScore(int subject) {
        this.studentScores[subject] = studentScores[subject];

        return studentScores[subject];
    }

    public void setScore(int studentScore, int subject) {
        studentScores[subject] = new Score(studentScore);
        subject++;
    }
}

File: Python/Student Grade Manager/src/Student.py
from Subject import Subject

class Student:
    def __init__(self, name):
        self.name = name
        self.scores = []

    def get_name(self):
        return self.name

    def get_student_average_score(self):
        return sum(self.scores) / len(self.scores)

    def print_report_card(self):
        print("Student Name: " + self.name +
              "Average score: " + str(self.get_student_average_score()))

